### PR TITLE
several validate infra failures fixes

### PR DESCRIPTION
**Pull request by YuvHayun**
## Description
- Fixed an issue where validate failed due to failure with relative and abs paths formatting.
- Fixed an issue where validate failed when didn't manage to create a reference to old content object.
- Fixed an issue where validate failed when receiving both -g & -i flags.

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/base_validator.py [hunk 1]
```diff
@@ -205,7 +205,11 @@ class InvalidContentItemResult(BaseResult, BaseModel):
 
     @property
     def format_readable_message(self):
-        return f"{str(self.path.relative_to(CONTENT_PATH))}: [{self.error_code}] - {self.message}"
+        # return f"{str(self.path.relative_to(CONTENT_PATH))}: [{self.error_code}] - {self.message}"
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/validators/base_validator.py:208]:
```suggestion
```

File: demisto_sdk/commands/validate/initializer.py [hunk 2]
```diff
@@ -482,9 +482,15 @@ def git_paths_to_basecontent_set(
...
-                        obj.old_base_content_object = BaseContent.from_path(
-                            old_path, git_sha=git_sha, raise_on_exception=True
-                        )
+                        try:
+                            obj.old_base_content_object = BaseContent.from_path(
+                                old_path, git_sha=git_sha, raise_on_exception=True
+                            )
+                        except (NotAContentItemException, InvalidContentItemException):
+                            logger.debug(
+                                f"Could not parse the old_base_content_object for {str(obj.path)}, setting a copy of the object as the old_base_content_object."
```
Comment (GuyAfik) [inline demisto_sdk/commands/validate/initializer.py:491]:
```suggestion
                                f"Could not parse the old_base_content_object for {obj.path}, setting a copy of the object as the old_base_content_object."
```

when using f-string it automatically calls `__str__` of the object

File: demisto_sdk/commands/validate/validators/base_validator.py [hunk 3]
```diff
@@ -205,7 +205,10 @@ class InvalidContentItemResult(BaseResult, BaseModel):
 
     @property
     def format_readable_message(self):
-        return f"{str(self.path.relative_to(CONTENT_PATH))}: [{self.error_code}] - {self.message}"
+        path: Path = self.path
+        if path.is_absolute():
+            path = path.relative_to(CONTENT_PATH)
+        return f"{str(path)}: [{self.error_code}] - {self.message}"
```
Comment (GuyAfik) [inline demisto_sdk/commands/validate/validators/base_validator.py:211]:
```suggestion
        return f"{path}: [{self.error_code}] - {self.message}"
```

**Review by GuyAfik — APPROVED** [review]
@YuvHayun add UT maybe?

**Pull request merged**